src/lib/openapi/spec/feature-search-query-parameters.ts

```diff
@@ -71,10 +71,10 @@ export const featureSearchQueryParameters = [
         name: 'limit',
         schema: {
             type: 'string',
-            example: '10',
+            example: '50',
         },
         description:
-            'The number of results to return in a page. By default it is set to 50',
+            'The number of feature environments to return in a page. By default it is set to 50.',
```

Comment:
We're explicit about it now. We can fix the limit to reflect the number of features not feature environments but:
* it's not trivial
* it's nice to not return too many results when someone has 10+ environments. With the current approach we have a natural limit for envs.